pkg/routeagent_driver/handlers/kubeproxy/vxlan.go

```diff
@@ -271,12 +274,21 @@ func (kp *SyncHandler) createVxLANInterface(activeEndPoint string, ifaceType int
 			}
 		}
 
+		// Revisit: This is a temporary work-around to fix https://github.com/submariner-io/submariner/issues/2422
+		// Allow the vx-submariner interface to get initialized.
+		time.Sleep(100 * time.Millisecond)
+
 		err = kp.netLink.EnableLooseModeReversePathFilter(VxLANIface)
 		if err != nil {
-			return errors.Wrap(err, "error enabling loose mode")
+			return errors.Wrapf(err, "error enabling loose mode on iface %q", VxLANIface)
+		}
+
+		_, err = kp.ensureLooseModeIsConfigured(VxLANIface)
+		if err != nil {
+			return errors.Wrap(err, "error while validating loose mode")
```

Comment:
Let’s just call `ensureLooseModeIsConfigured`, it ends up doing the same as the initial wait and `EnableLooseMode`.

Reply:
I thought it would be easier to read with the comment that we are waiting for the interface to get initialised, but anyways, I can change it. Thanks for reviewing.